core/src/main/scala/akka/kafka/ConsumerSettings.scala

```diff
@@ -299,6 +299,26 @@ class ConsumerSettings[K, V] @InternalApi private[kafka] (
   def withGroupInstanceId(groupInstanceId: String): ConsumerSettings[K, V] =
     withProperty(ConsumerConfig.GROUP_INSTANCE_ID_CONFIG, groupInstanceId)
 
+  /**
+   * A list of class names or class types, ordered by preference, of supported
+   * partition assignment strategies that the client will use to distribute
+   * partition ownership amongst consumer instances when group management is used.
+   *
+   * See https://kafka.apache.org/documentation/#consumerconfigs_partition.assignment.strategy
+   */
+  def withPartitionAssignmentStrategy(strategy: String): ConsumerSettings[K, V] =
```

Comment:
Might be nice to have a convenience method which takes a collection (maybe an array?) of strings and builds the property string to pass here?